src/db/entities/coinPrice.ts

```diff
@@ -0,0 +1,14 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+import { Coin } from './coin';
+
+@Entity()
+export class CoinPrice {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column(() => Coin)
+  coin!: Coin;
+
+  @Column({ type: 'decimal' })
```

Comment:
Why decimal?
Don't we need floating numbers?

Reply:
yes, sorry. edited.

Reply:
Did you push your code? :)
It does not seem to be changed.

Reply:
No. after fixing all the comments I'll push it.